modules/managing-memcached-operator-using-olm.adoc

```diff
@@ -46,34 +52,56 @@ for more information on manually defining a manifest file.
 
 . *Deploy the Operator.*
 
-.. Deploy an Operator by applying the Operator’s manifest to the desired namespace
-in the cluster:
+.. Create an OperatorGroup that specifies the namespaces that the Operator will
```

Comment:
Nit: 
```suggestion
.. Create an `OperatorGroup` that specifies the namespaces that the operator will
```

Reply:
Thanks for the suggestion but we actually do capitalize Operator and don't use markup as per our guidelines [1].

[1] https://github.com/openshift/openshift-docs/blob/enterprise-4.1/contributing_to_docs/doc_guidelines.adoc#api-object-formatting